app/src/components/common/TextInput.vue

```diff
@@ -52,6 +55,10 @@ input {
   border-radius: 4px;
 }
 
+input:invalid {
```

Comment:
check the styles above please, there are duplicates

Reply:
Removed duplicates in the recent commit!